samples/browser/src/main/java/org/mozilla/samples/browser/ExternalAppBrowserFragment.kt

```diff
@@ -101,6 +107,18 @@ class ExternalAppBrowserFragment : BaseBrowserFragment(), UserInteractionHandler
         return layout
     }
 
+    override fun onResume() {
+        super.onResume()
+        val currTime = SystemClock.elapsedRealtimeNanos()
+        emitForegroundTimingFact(currTime)
```

Comment:
Quick breakdown of our browsers:
- Sample Browser (this one): Demo of our Android Components
- Reference Browser: Starting point for building a full browser with Android Components
- Fenix: some product idk

All 3 have their own browser fragments and external app browser fragments. If you want to add telemetry for Fenix it should go in the Fenix codebase :)

Reply:
😰 dang this is confusing hahahaha